lib/rules/no-unused-prop-types.js

```diff
@@ -44,6 +44,9 @@ module.exports = {
         },
         skipShapeProps: {
           type: 'boolean'
+        },
+        extraPropsFunc: {
```

Comment:
i think this would be better as an array of unique strings - that way multiples could be provided.

Let's also rename it to "propWrapperFunctions"